draft-ietf-rats-msg-wrap.md

```diff
@@ -246,6 +254,28 @@ func CMWTypeSniff(b []byte) (CMW, error) {
 }
 ~~~
 
+## CMW Collections {#cmw-coll}
+
+Layered attesters and composite devices ({{Sections 3.2 and 3.3 of -rats-arch}}) generate evidence that consists of multiple parts.
+
+For example, in data center servers, it is not uncommon for separate Attesting Environments (AE) to serve a subsection of the entire machine.
+One AE might produce Evidence including Claims that represent what was booted on the main CPU, or on a SmartNIC plugged into a PCIe slot, or on the machine's GPU.
+
+To address the composite Attester use case, this document defines a CMW "collection" as a container that holds several CMW items, each with a label that is unique within the scope of the collection.
+
+The CMW collection ({{fig-cddl-collection}}) is defined as a CBOR map or JSON object with CMW values.
+The position of a `cmw` entry in the `cmw-collection` is not significant.
+Instead, the labels identify a conceptual message that, in the case of a composite attester, should typically correspond to a component of a system.
+Labels can be strings or integers that serve as a mnemonic for different conceptual messages in the collection.
```

Comment:
As TE cannot produce Conceptual Messages this would mean you always mean AE?

Reply:
The ambiguous part is the meaning of 'correspond'. Either the label identifies the AE that produced the evidence, or it identifies the TEs that are the subject of claims collection.